### DISCUSSION TITLE
input type = search

### DISCUSSION THREAD

**Discussion by jaeyeong815** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#37] [General]
[mozilla site](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search)

부끄럽지만 input type에 search가 있는 줄 몰랐네요 ^^;
맨날 text나 password, email만 쓰다가..!
사이트 확인해보니 검색 필드에 포함된 값을 나타내는 문자열이 포함되는 속성이 있네요!
볼드표시 기능 구현할 때 요긴하게 쓰일 듯 합니다 ~ 물론 검색 기능 구현할때도요!

다들 알고 계실 수 있겠지만 .. 저를 위해 기록해둡니다 하핫

**Comment by baechanghyeon** [comment]
첨부해주신 링크 타고 들어가니까 input에 다양한 type 종류가 있었네요!! ㅎㅎ 보고 잘 참고하겠습니다! 감사합니다 :)
